content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -332,16 +330,13 @@ $ tree .dvc/cache
     └── 0b40427ee0998e9802335d98f08cd98f
 ```
 
-The cache file with `.dir` extension is a special text file that contains the
-mapping of files in the `data/` directory (as a JSON array), along with their
-hash values. The other two cache files are the files inside `data/`.
+This `.dir` file contains the mapping of files in `data/images` (as a JSON
+array), including their hash values. That's how DVC knows that the other two
+cached files belong in the directory:
 
-A typical `.dir` cache file looks like this:
-
-```dvc
-$ cat .dvc/cache/19/6a322c107c2572335158503c64bfba.dir
-[{"md5": "dff70c0392d7d386c39a23c64fcc0376", "relpath": "cat.jpeg"},
-{"md5": "29a6c8271c0c8fbf75d3b97aecee589f", "relpath": "index.jpeg"}]
+```json
+[{ "md5": "dff70c0392d7d386c39a23c64fcc0376", "relpath": "cat.jpeg" },
+{ "md5": "29a6c8271c0c8fbf75d3b97aecee589f", "relpath": "index.jpeg" }]
 ```
```

Comment:
Restyled changes this (see https://github.com/iterative/dvc.org/pull/1902#pullrequestreview-521161503). We can just ignore Restyled and merge this anyway right?

> It may be a autoformatted later in another PR though...

Reply:
No, I don't think so. If it's important enough removing `json` prefix should help.

Reply:
Hmmm rather keep `json` so there's highlighting. OK I'll let it format the block.

Reply:
But it can be misleading then ....

Reply:
We can keep the command that we had `cat ...`

Reply:
OK, I reverted to `$ cat` but I think it looks better with the JSON formatting.